admin/tox-build-extensions.py

```diff
@@ -0,0 +1,16 @@
+"""
```

Comment:
This script is not needed at all.  
By default, tox calls **setup.py bdist_wheel** which builds the extensions.